Baconit/Strings/es-es/Resources.resw

```diff
@@ -0,0 +1,889 @@
+    <value>Perfil</value>
+  </data>
+  <data name="progressTextTB.Text" xml:space="preserve">
+    <value>Cargando</value>
+  </data>
+  <data name="ProTip.Text" xml:space="preserve">
+    <value>Pista</value>
+  </data>
+  <data name="QuickReviewCT.Content" xml:space="preserve">
+    <value>Deja un comentario rápido</value>
+  </data>
+  <data name="quickSearchBox.PlaceholderText" xml:space="preserve">
+    <value>Busca cualquier cosa</value>
+  </data>
+  <data name="RedduntAccountTB.Text" xml:space="preserve">
+    <value>Desbloquea más contenido de Reddunt al añadir una cuenta.Las cuentas de Reddit son anónomas,simples y gratuitas - y Reddunt las mantendrá así.</value>
+  </data>
+  <data name="SearchASUPTBS.PlaceholderText" xml:space="preserve">
+    <value>Busca cualquier cosa; subreddits, usuarios, post, etc.</value>
+  </data>
+  <data name="SearchForTB.Text" xml:space="preserve">
+    <value>Buscar:</value>
+  </data>
+  <data name="SearchTBs.Text" xml:space="preserve">
+    <value>Buscar</value>
+  </data>
+  <data name="searchTextBlock.Text" xml:space="preserve">
+    <value>Buscar</value>
+  </data>
+  <data name="SendingCommentTB.Text" xml:space="preserve">
+    <value>Enviando comentario...</value>
+  </data>
+  <data name="settingsGridTB.Text" xml:space="preserve">
+    <value>Configuración</value>
+  </data>
+  <data name="ShareHeartTB.Text" xml:space="preserve">
+    <value>Comparte el ❤</value>
+  </data>
+  <data name="ShowMoreTB.Text" xml:space="preserve">
+    <value>Mostrar más</value>
+  </data>
+  <data name="signInGridTB.Text" xml:space="preserve">
+    <value>Entrar</value>
+  </data>
+  <data name="subscribersTextBlock.Text" xml:space="preserve">
+    <value>73,000 subscriptores</value>
+  </data>
+  <data name="textBlock.Text" xml:space="preserve">
+    <value>¡Destripe!</value>
+  </data>
+  <data name="TrendingSubreddits.Text" xml:space="preserve">
+    <value>Tendencias en Subreddits</value>
+  </data>
+  <data name="TryItTip.Text" xml:space="preserve">
+    <value>¡Adelante! ¡Vuelve a intentarlo!</value>
+  </data>
+  <data name="ViewCommentTip.Text" xml:space="preserve">
+    <value>Para ver los comentarios para este o cualquier por, pon el puntero del ratón o el dedo en la barra del título y navega hacia abajo.</value>
+  </data>
+  <data name="WelcomeText.Text" xml:space="preserve">
+    <value>Bienvenido Quinbd</value>
+  </data>
+  <data name="AccountAgeTB.Text" xml:space="preserve">
+    <value>0 días de antiguedad</value>
+  </data>
+  <data name="AllTime.Text" xml:space="preserve">
+    <value>Todo</value>
+  </data>
+  <data name="CakeDayTB.Text" xml:space="preserve">
+    <value>¡Día del pastel!</value>
+  </data>
+  <data name="CommentSort.Text" xml:space="preserve">
+    <value>Ordenar:</value>
+  </data>
+  <data name="CommentSortFMIBest.Text" xml:space="preserve">
+    <value>Lo mejor</value>
+  </data>
+  <data name="CommentSortFMIContro.Text" xml:space="preserve">
+    <value>Controvérsico</value>
```

Comment:
I would use "Polémico", "Polémica" or "Controversial" instead of "Controvérsico".
I'd prefer polémico or polémica, but it depends on the gender of the subject/people. You could use Controversial and it would do in Spanish easier than polémico because "controversial" doesn't depend on the gender.